CorsixTH/Lua/objects/reception_desk.lua

```diff
@@ -91,11 +91,15 @@ function ReceptionDesk:onClick(ui, button)
   end
 end
 
+local queue_rerouted = false
```

Comment:
What is this extra bit of state doing out here on it's own. It's not part of the reception desk so it's global. Doesn't necessarily play nicely with save state, and I don't see why it's needed.

If it is in fact needed, but it on the desk.

Reply:
If we don't have a flag... we execute "rerouteAllPatients()" about 5-10 times before it stops. The flag means it only happens once. I'll move it inside the desk itself.